Tools/TestWebKitAPI/Configurations/TestWebKitAPIBase.xcconfig

```diff
@@ -144,12 +144,17 @@ OTHER_LDFLAGS_PLATFORM_cocoatouch = -framework WebCore -framework UIKit -framewo
 
 LD_RUNPATH_SEARCH_PATHS = "@loader_path";
 
-MAC_CODE_SIGN_ENTITLEMENTS = $(MAC_CODE_SIGN_ENTITLEMENTS_$(USE_INTERNAL_SDK));
-MAC_CODE_SIGN_ENTITLEMENTS_ = Configurations/TestWebKitAPI-macOS.entitlements;
-MAC_CODE_SIGN_ENTITLEMENTS_YES = Configurations/TestWebKitAPI-macOS-internal.entitlements;
+CODE_SIGN_INJECT_BASE_ENTITLEMENTS = NO;
 
-CODE_SIGN_ENTITLEMENTS[sdk=embedded*] = Configurations/TestWebKitAPI-iOS.entitlements;
-CODE_SIGN_ENTITLEMENTS[sdk=macosx*] = $(MAC_CODE_SIGN_ENTITLEMENTS);
+OTHER_CODE_SIGN_FLAGS = --entitlements $(WK_PROCESSED_XCENT_FILE);
+OTHER_CODE_SIGN_FLAGS[sdk=*simulator] = --entitlements $(WK_PROCESSED_BASE_XCENT_FILE);
+OTHER_LDFLAGS_ENTITLEMENTS[sdk=*simulator] = -Xlinker -sectcreate -Xlinker __TEXT -Xlinker __entitlements -Xlinker $(WK_PROCESSED_XCENT_FILE);
+
+WK_PROCESSED_BASE_XCENT_FILE = ;
+WK_PROCESSED_BASE_XCENT_FILE[sdk=*simulator] = $(TEMP_FILE_DIR)/$(FULL_PRODUCT_NAME).xcent;
```

Comment:
naming nit: I wonder if it would be clearer to give this a simulator-specific name, since its purpose is to hold entitlements that apply to the code signature of simulated binaries only. how about `WK_SIMULATED_XCENT_FILE`?